### ISSUE TITLE
[Bug] Circular shape abnormality in editing state

### ISSUE THREAD

**Issue by Thunder7991** [JamesLMilner/terra-draw#237]
**Describe the bug**
Hello @JamesLMilner  , in the recent version update, there have been significant changes to the circle, and the mode of scaling according to the center has been removed. I feel that doing so is very regrettable.
Circular editing should be scaled according to the center.
Secondly, an issue was found in the latest version. After multiple scaling, the position of the circular point is abnormal. See screenshot
**Terra Draw npm version**
Latest version

**Expected behavior**
center-fix and Prototype graphics editing is normal

**Screenshots**
![1712470124123](https://github.com/JamesLMilner/terra-draw/assets/61643801/9dec28dc-2936-4a84-b778-cde697d3ddc4)

![1712470140205](https://github.com/JamesLMilner/terra-draw/assets/61643801/4d740e5e-b456-4065-8414-1058c3426516)

**Desktop (please complete the following information):**
 - OS: [windows]
 - Browser [e.g. chrome]
 - Version [Latest ]

**Comment by Thunder7991** [comment]
Hello @JamesLMilner , the latest Terra Draw has undergone significant updates in circular drawing. But abandoning the center fix mode doesn't feel very convenient during use. Can it be added?, Or I can try adding relevant methods.

**Comment by JamesLMilner** [comment]
Hey @Thunder7991 can you explain what you mean by 'enter fix mode doesn't feel very convenient during use'? The modes were renamed to reflect that they are based on the web mercator projection.

You can use: 

```javascript
resizable: "center-web-mercator",
```

Or:

```javascript
center-fixed-web-mercator
```

If those aren't what you meant can you explain more?

**Comment by Thunder7991** [comment]
This is the case where the editing status is reduced to the minimum and repeatedly zoomed in. I guess it is a matter of calculation.

**Comment by JamesLMilner** [comment]
Sorry to hear you are having problems. Unfortunately I don't think I fully follow what you are saying. The last update made a number of changes that improved the resizing behaviours.

If you want to have a fixed circle that resizes, you can do it like so:

```javascript
resizable:: 'center-fixed-web-mercator'
```

That results in this behaviour:

https://github.com/JamesLMilner/terra-draw/assets/8822075/880e7a9d-b2a5-4753-8f39-4571b1174367

The issue of the shape getting deformed at small sizes is one I know of and am aiming to fix as part of https://github.com/JamesLMilner/terra-draw/issues/239 - with this you will be able to provide a function, such as minimum area/width/height etc to prevent deformation.

**Comment by JamesLMilner** [comment]
I'm just looking through the issue tracker - this issue feels like a duplicate of https://github.com/JamesLMilner/terra-draw/issues/230 - is that right? If so, I will close that issue in favour of this one.

**Comment by Thunder7991** [comment]
I'm very sorry, it was due to my negligence in configuration. However, there may still be graphics anomalies when zooming in and out, as shown in the video below:

https://github.com/JamesLMilner/terra-draw/assets/61643801/31a2164a-b7f4-4a29-ba82-481cee9c7fdd

**Comment by JamesLMilner** [comment]
I think this issue can be resolved as described in my comment for #230, using the new `validation` flag, passing a function to validate the circle is well formed/of a minimum size.
